v5-MIGRATION.md

```diff
@@ -15,3 +15,4 @@
 - `getSubscriptionSkusWith` and `getNonSubscriptionSkusWith` now receive `storeProducts` instead of `skus`.
 - `purchaseProductWith` now accepts a `StoreProduct` instead of a`SkuDetails`
 - `PurchaserInfo` has been renamed to `CustomerInfo`
+- `ReceiveOfferingsListener` migrated to `ReceiveOfferingsCallback`
```

Comment:
are we missing the purchaseserrorcallback here?

Reply:
yes, good catch. I am preparing another PR with the migration guide, I will add it there

Reply:
https://github.com/RevenueCat/purchases-android/pull/401/files#diff-486f832ab9514b7472efb6fb85828d2a47c1458c6a2aed6d3dbde1cda2eb2b75 I added it here